index.html

```diff
@@ -0,0 +1,180 @@
+                    <div class="contests-title">Contests</div>
+                    <div class="contests">
+                        <div class="contest from-left">
+                            <img src="/Assets/images/pcpc2023.jpeg" alt="PCPC23" class="contest-img">
+                            <div class="contest-info">
+                                <h3>Palestinian Collegiate Programming Contest (PCPC)</h3>
+                                <p>Participated in PCPC22 and PCPC23, securing 7th place in PCPC23, which qualified us for the Arab Collegiate Programming Contest (ACPC) in Egypt. This achievement enhanced our problem-solving and teamwork skills through high-level national and regional competitions.</p>
+                            </div>
+                        </div>
+                        <div class="contest from-right">
+                            <img src="/Assets/images/bits&bytes.jpeg" alt="Bits&Byte" class="contest-img">
+                            <div class="contest-info">
+                                <h3>Bits&Byte Competition</h3>
+                                <p>Won 1st place in the Bits&Byte coding competition, excelling in algorithm optimization and problem-solving.</p>
+                            </div>
+                            <img src="/Assets/icons/badge-icon.svg" class="contest-icon">
+                        </div>
+                        <div class="contest from-left">
+                            <img src="/Assets/images/ieeeXtreem.jpeg" alt="IEEEXtreme" class="contest-img">
+                            <div class="contest-info">
+                                <h3>IEEEXtreme 16.0 & 17.0</h3>
+                                <p>Participated in IEEEXtreme 16.0 and served as an ambassador for IEEEXtreme 17.0, where I also achieved 3rd place locally. As an ambassador, I promoted the contest and supported participants, all while demonstrating strong coding abilities under time constraints.</p>
+                            </div>
+                            <img src="/Assets/icons/trophy-icon.svg" class="contest-icon">
+                        </div>
+                        <div class="contest from-right">
+                            <img src="/Assets/images/U-tech-league.jpeg" alt="U-TECH League" class="contest-img">
+                            <div class="contest-info">
+                                <h3>U-TECH League</h3>
+                                <p>Secured 1st place in the U-TECH League AI contest by developing a chatbot for JAWWAL company. I contributed as a front-end developer, focusing on creating an intuitive and user-friendly interface for the chatbot application.</p>
+                            </div>
+                            <img src="/Assets/icons/web-icon.svg" class="contest-icon">
+                        </div>
+                    </div>
+                </div>
+            </section>
+        </main>
+        <footer role="contentinfo" class="footer">
```

Comment:
You dont need to add `contentinfo` for the footer because the default role is `contentinfo`. You can read this for more information https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/contentinfo_role.

Reply:
fixed